frontend/src/templates/Field/ChildrenCompound/ChildrenCompoundField.tsx

```diff
@@ -0,0 +1,404 @@
+      description += ` You can add another child if you'd like by clicking the "Add another child" button below`
+    }
+
+    return description
+  }, [fields.length, schema.allowMultiple])
+
+  const numChild = fields.length ?? 0
+
+  return (
+    <FieldContainer
+      schema={schema}
+      {...fieldContainerProps}
+      errorKey={childrenInputName}
+    >
+      <VisuallyHidden id={`children-desc-${schema._id}`}>
+        {ariaChildrenDescription}
+      </VisuallyHidden>
+      <VStack
+        spacing={6}
+        align="stretch"
+        aria-describedby={`children-desc-${schema._id}`}
+        aria-labelledby={`${schema._id}-label`}
+      >
+        <VStack align="stretch" role="list">
+          {fields.map((field, currChildBodyIdx) => (
+            <ChildrenBody
+              key={`body-${currChildBodyIdx}`}
+              {...{
+                currChildBodyIdx,
+                schema,
+                fields,
+                field,
+                colorTheme,
+                remove,
+                myInfoChildrenBirthRecords,
+                isSubmitting,
+                formContext,
+              }}
+            />
+          ))}
+        </VStack>
+        {schema.allowMultiple ? (
+          <HStack>
+            <Button
+              isDisabled={isSubmitting}
+              alignSelf="start"
+              leftIcon={<BiPlus />}
+              aria-label="Add another child"
+              onClick={() => {
+                append([''])
+              }}
+            >
+              Add another child
+            </Button>
+            <Spacer />
+            <Text
+              textStyle="body-2"
+              color="secondary.400"
+            >{simplur`${numChild} child[|ren] added`}</Text>
+          </HStack>
+        ) : null}
+      </VStack>
+    </FieldContainer>
+  )
+}
+
+interface ChildrenBodyProps {
+  currChildBodyIdx: number
+  schema: ChildrenCompoundFieldSchema
+  fields: FieldArrayWithId<
+    ChildrenCompoundFieldInputs,
+    `${string}.child`,
+    'id'
+  >[]
+  field: FieldArrayWithId<ChildrenCompoundFieldInputs, `${string}.child`, 'id'>
+  colorTheme: FormColorTheme
+  remove: UseFieldArrayRemove
+  myInfoChildrenBirthRecords?: MyInfoChildData
+  isSubmitting: boolean
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  formContext: UseFormReturn<ChildrenCompoundFieldInputs, any>
+}
+
+const ChildrenBody = ({
+  currChildBodyIdx,
+  schema,
+  fields,
+  field,
+  colorTheme,
+  remove,
+  myInfoChildrenBirthRecords,
+  isSubmitting,
+  formContext,
+}: ChildrenBodyProps): JSX.Element => {
+  const { register, getValues, setValue, watch } = formContext
+
+  const childNamePath = useMemo(
+    () => `${schema._id}.child.${currChildBodyIdx}.0`,
+    [schema._id, currChildBodyIdx],
+  )
+  const { onChange: selectOnChange, ...selectRest } = register(childNamePath)
+
+  const childName = watch(childNamePath) as unknown as string
+
+  const allChilds = useMemo<string[]>(() => {
```

Comment:
/nit funny plural? 😅 

```suggestion
  const allChildren = useMemo<string[]>(() => {
```